atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -49,10 +49,10 @@
 import org.apache.commons.pool2.impl.DefaultEvictionPolicy;
 import org.apache.commons.pool2.impl.GenericObjectPool;
 import org.apache.commons.pool2.impl.GenericObjectPoolConfig;
-import org.apache.thrift.transport.TFramedTransport;
 import org.apache.thrift.transport.TMemoryInputTransport;
 import org.apache.thrift.transport.TTransport;
 import org.apache.thrift.transport.TTransportException;
+import org.apache.thrift.transport.layered.TFramedTransport;
```

Comment:
This would be an ABI break 😬 let's check if there are transitive users of `libthrift`

Reply:
turns out there are! we might want to shade it instead, open to discussion / suggestions

Reply:
I think we may have to force this break to folks transitively consuming libthrift from atlasdb (I'd be surprised if folks are directly using `TFramedTransport` that moved packages). We probably need to make the same bump & changes in https://github.com/palantir/cassandra (@leonz & @Sam-Kramer for SA)

Reply:
agreed on forcing the break